src/main/java/tars/Parser.java

```diff
@@ -0,0 +1,95 @@
+package tars;
+
+import java.util.ArrayList;
+/**
+ * Processes User inputs and executes commands based on inputs
+ * Has method to also list all Tasks added by User
+ *
+ * @author csk
+ * @version 1
+ */
+
+public class Parser {
+    private static final String LINE = "    _____________________________________________";
+
+    public Parser() {
+    }
+    /**
+     * Checks user inputs after being taken as string and string[] variables
+     * Chooses appropriate methods to call based on input given or throws error message
+     * depending on incorrect input received
+     *
+     * @param entryParts
```

Comment:
you could write a short phrase to describe the params and return value for clarity
e.g. @param entry The input string from the user
@param tasks The list of tasks